src/reducers/viewHandler.js

```diff
@@ -0,0 +1,381 @@
+import { Map } from 'immutable';
```

Comment:
can u use iMap instead of Map ? Let's use this across the app .  It will also fix this issue in codacy "A function with a name starting with an uppercase letter should only be used as a constructor." so it's a win-win.

Reply:
Also iMap is simple to reason about and we will know it's the immutable one